### DISCUSSION TITLE
Time frame for 4.0 ?

### DISCUSSION THREAD

**Discussion by c1972** [thorsten/phpMyFAQ#3032] [Q&A] [answered]
Hey Thorsten,

it seems that **you mostly work alone** on this project. Sadly not that much from other users. or?

Anyway: what do you expect when 4.0 will be availlable?

Thanks + regards Christian

**Comment by thorsten** [comment, marked as answer]
@modelrailroader helps a lot for a year now, but yes, mainly I work alone on this project for more that 23 years now. :-)

I hope we get 4.0.0-alpha.3 with new template engine for the frontend done during summer and we can start the beta phase. The good thing it's quite stable already.
